src/checkbox/checkbox.js

```diff
@@ -15,255 +15,148 @@
  * limitations under the License.
  */
 
-(function() {
-  'use strict';
+// Numeric constants used in this component.
+const numbers_ = Object.freeze({
+  TINY_TIMEOUT: 0.001
+});
+
+// CSS classes used in this component.
+const classes_ = Object.freeze({
```

Comment:
Any particular reason for going with `this.classes_ = Object.freeze({});` rather than `MaterialCheckbox.prototype.classes_ = Object.freeze({});` ?